### DISCUSSION TITLE
Apex test Class Result Better UI

### DISCUSSION THREAD

**Discussion by sc5148** [forcedotcom/salesforcedx-vscode#5765] [Ideas]
The apex test result output is not at all user friendly. Please implement these for a better user experience: 
1. The output of the classes should be arranged alphabetically instead of displaying it randomly. 
2. There should be an option to disable displaying the "uncovered lines" column. This messes up the entire format if word wrap is enabled in the terminal.

**Comment by lcampos** [comment]
Hi @sc5148 , is this specific to the test result output in the terminal or somewhere else in VSCode (e.g. test sidebar) ?

**Comment by sc5148** [comment]
@lcampos I am talking about the test result output in the **terminal**. In the sidebar it's already arranged alphabetically so no issues.

**Comment by sc5148** [comment]
@lcampos Any update on this one?

**Comment by lcampos** [comment]
@sc5148 we're currently working on enhancements to the apex:test:run & apex:test:report commands but modifying the output format will be one of the last things we tackle. We are focusing on fixing issues with incorrect/incomplete information being reported and performance enhancements.
cc @ntotten @smaddox-sf
